File: src/main/java/thirtytwo/degrees/web/IndexController.java
package thirtytwo.degrees.web;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;

import java.io.IOException;
import java.io.Writer;

@Controller
public class IndexController {

    @RequestMapping(value = "/")
    public String home() throws IOException {
        return "forward:/index.html";
    }
}

File: scala_2.9.1/src/main/scala/thirtytwo/degrees/halfpipe/scalaexample/web/IndexController.scala
package thirtytwo.degrees.halfpipe.scalaexample.web

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping

/**
 * User: gibbsb
 * Date: 9/22/12
 * Time: 4:17 PM
 */
@Controller
class IndexController {

  @RequestMapping(Array("/"))
  def home() = "forward:/index.html"
}
